src/api/job.js

```diff
@@ -0,0 +1,56 @@
+const {Router} = require('express');
+const Job = require('../persistence/job');
+
+const router = new Router();
+
+// Admin Create a new job with post method
+router.post('/', async (request, response) => {
+  try {
+    const {
+      title,
+      salaryrange,
```

Comment:
If the variable name contains more than 1 word, we should use the camel case format.

Reply:
Updated.